src/settings/_motion.scss

```diff
@@ -0,0 +1,27 @@
+// Settings for the SUI Motion Tokens
+
+// Duration Tokens --------------------
+// Naming & timing reference: Polaris Design System https://polaris-tokens.herokuapp.com/
+$duration-none: 0 !default;
+$duration-fast: 100ms !default;
+$duration-base: 200ms !default;
+$duration-slow: 300ms !default;
+$duration-slower: 400ms !default;
```

Comment:
I'd suggest we simplify to 4 options: none, normal, fast and slow (easy to remember and to pick from)
- 0
- 100
- 300
- 500